recipes/odbc/all/test_package/CMakeLists.txt

```diff
@@ -2,7 +2,7 @@ cmake_minimum_required(VERSION 3.1)
 project(test_package C)
 
 include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
-conan_basic_setup()
+conan_basic_setup(TARGETS)
```

Comment:
Quick question if you don't mind.
Why is this change needed? Wouldn't the following `find_package()` set the targets?

Reply:
It's orthogonal. The purpose here is to not include headers folders through conan_basic_setup().